packages/devextreme/testing/tests/DevExpress.ui.widgets/splitter.tests.js

```diff
@@ -2537,6 +2537,24 @@ QUnit.module('Visibility of control elements', {
 });
 
 QUnit.module('Events', moduleConfig, () => {
+    QUnit.test('should output correct size value for panes in onResizeEnd event (T1240125)', function(assert) {
+        const resizeHandlerStub = sinon.stub().callsFake(function(eventArgs) {
```

Comment:
[excess code]
You actually don't need a stub here. You can just pass a function directly

Reply:
[85ecf61](https://github.com/DevExpress/DevExtreme/pull/27793/commits/85ecf6149775d598cf2b711ea1ec5f599bb52b25)